### PR TITLE
[Doc] Add doc for SQS permission

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/streamnative/pulsar-io-sqs/issues/124

### CODE REVIEW

File: docs/sqs-sink.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Pull the SQS connector Docker image from [here](https://hub.docker.com/r/streamn
 
 Before using the SQS sink connector, you need to configure it. Below are the properties and their descriptions.
 
-You can create a configuration file (JSON or YAML) to set the following properties.
+You can create a configuration file (JSON or YAML) in the `conf` folder to set the following properties.
```
Comment (Anonymitaet) [inline docs/sqs-sink.md:47]: @freeznet is it correct?

**Comment by Anonymitaet** [comment]
@freeznet Could you please help review this PR? Thanks

File: docs/sqs-sink.md (hunk 1 above)
Comment (freeznet) [reply docs/sqs-sink.md:47]: @Anonymitaet I dont think the conf file must be in the `conf` folder.

Comment (Anonymitaet) [reply docs/sqs-sink.md]: @freeznet OK, I've reverted the changes, PTAL.

**Review by freeznet — APPROVED** [review]
LGTM

**Pull request merged**